libs/hdf-converters/src/burpsuite-mapper.ts

```diff
@@ -64,14 +64,6 @@ function nistTag(input: string): string[] {
   return CWE_NIST_MAPPING.nistFilter(cwe, DEFAULT_NIST_TAG).concat(['Rev_4']);
 }
 
-function parseXml(xml: string): Record<string, unknown> {
-  const options = {
-    attributeNamePrefix: '',
-    textNodeName: 'text',
-    ignoreAttributes: false
-  };
-  return parser.parse(xml, options);
-}
```

Comment:
You moved the parseXml? I see that it is imported.

Realized that it was repeated over in all other parsers.